Doc/library/pathlib.rst

```diff
@@ -636,6 +636,22 @@ Concrete paths provide the following methods in addition to pure paths
 methods.  Many of these methods can raise an :exc:`OSError` if a system
 call fails (for example because the path doesn't exist):
 
+.. method:: Path.absolute()
```

Comment:
saw the discussion on the linked issue, and I think it's better to pluck it from this one. Doing that on next commit.